tests/unit/test_utils.py

```
@@ -346,50 +347,62 @@ def test_incomplete_sequencing_runs_are_skipped(self):
+        open(os.path.join(run_dir, "RunInfo.xml"), "w").close()
+        open(os.path.join(run_dir, "CopyComplete.txt"), "w").close()
         copyfile(
             os.path.join(TEST_DATA_DIR, "example_samplesheet.csv"),
-            os.path.join(old_run, "samplesheet.csv"),
+            os.path.join(run_dir, "samplesheet.csv"),
         )
 
-        # update the modified time of RunInfo.xml to be older than the
-        # given max age (1621091308 => 16:08:28 - 15/5/2021)
+        # update the modified time of RunInfo.xml to be 100 hours old
+        old_utime = int((datetime.now() - timedelta(hours=100)).timestamp())
```

Ruff (0.8.0)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)